src/Layout/ComputationMenu/YearSelect.tsx

```diff
@@ -9,30 +9,31 @@ export interface YearSelectProps {
 export class YearSelect extends React.Component<YearSelectProps> {
     render() {
         return (
-            <div className="form-group row">
-                <label className="text-left col-sm-2 col-form-label" htmlFor="year_select">
-                    År
+            <div className="field">
+                <label className="label" htmlFor="year_select">
+                    Valgt år
                 </label>
-                <div className="col-sm-10">
-                    <select
-                        title="Velg år"
-                        id="year_select"
-                        value={this.props.year}
-                        onChange={this.props.onYearChange}
-                        className="form-control"
-                        name="year"
-                    >
-                        {this.props.electionYears.map((item, index) => {
-                            return (
-                                <option
-                                    key={index} // By convention all children should have a unique key prop
-                                    value={item}
-                                >
-                                    {item}
-                                </option>
-                            );
-                        })}
-                    </select>
+                <div className="control">
+                    <div className="select is-dark is-fullwidth">
+                        <select
+                            title="Velg år"
+                            id="year_select"
+                            value={this.props.year}
+                            onChange={this.props.onYearChange}
+                            name="year"
+                        >
+                            {this.props.electionYears.map((item, index) => {
+                                return (
+                                    <option
+                                        key={index} // By convention all children should have a unique key prop
```

Comment:
This probably doesn't need to stay a comment.

Reply:
Removed the comment